core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -99,7 +99,7 @@ public class Adapter implements IAdapter, NamedBean {
 	private @Getter boolean autoStart = APP_CONSTANTS.getBoolean("adapters.autoStart", true);
 	private @Getter boolean replaceNullMessage = false;
 	private @Getter int messageKeeperSize = 10; //default length of MessageKeeper
-	private Level msgLogLevel = Level.toLevel(APP_CONSTANTS.getProperty("msg.log.level.default", "INFO"));
+	private @Getter Level msgLogLevel = Level.toLevel(APP_CONSTANTS.getProperty("msg.log.level.default", "INFO"));
```

Comment:
Er zit een test in het framework die kijkt of het Setter type overeenkomt met de Getter. Daar gaat hij nu fout op :S
Je kan het beste het veld hernoemen naar wat het is, `defaultMessageLogLevel` 😃 .